### DISCUSSION TITLE
Ente Password Manager & Ente VPN 😉

### DISCUSSION THREAD

**Discussion by haritvsmurali** [ente-io/ente#2851] [General]
Only Indian Brand focusing on building an Truly & Fully Private, Safe, Encrypted Ecosystem. Considering it's reputation in Privacy World, Ente should next focus on building PASSWORD MANAGER & VPN Apps as well.  🤟🏼😇

**Comment by dnred** [comment]
Definetly not right now.

They're still a small company and there are already great PMs and VPNs like Bitwarden or Proton Pass and Mullvad or Proton VPN or IVPN. And they haven't even released v1 of Photos yet.

Maybe somewhere **way** down the line but I don't expect it to be honest. If they're gonna expand it's not going to be a VPN or a PM, it'd probably be a drive service.

**Comment by ZethiaZ** [comment]
I feel like there are many services already doing those things well, and there's no advantage to bundling their own. Plus, as a customer I can say I actively dislike having too many things bundled together, but maybe that's just me.
